src/registry/FileSystemRegistry.ts

```diff
@@ -63,7 +66,15 @@ export class FileSystemRegistry extends SynchronousRegistry implements IRegistry
       warpRoutes[routeId] = filePath;
     }
 
-    return (this.listContentCache = { chains, deployments: { warpRoutes } });
+    const warpDeployConfigURIs: RegistryContent['deployments']['warpDeployConfigURIs'] = {};
+    const warpDeployFiles = this.listFiles(path.join(this.uri, this.getWarpRoutesPath()));
+    for (const filePath of warpDeployFiles) {
+      if (!WARP_ROUTE_DEPLOY_FILE_REGEX.test(filePath)) continue;
+      const routeId = warpRouteDeployConfigPathToId(filePath);
+      warpDeployConfigURIs[routeId] = filePath;
+    }
+    
+    return (this.listContentCache = { chains, deployments: { warpRoutes, warpDeployConfigURIs } });
```

Comment:
Nit: wondering why the `warpDeployConfigURIs` name feels inconsistent with the other exports here. Would `warpDeployConfigs` be better?

Reply:
true. Updated